CONTRIBUTING.md

```diff
@@ -63,6 +63,11 @@ make test
 make build-docs
 ```
 
+> *Note:* If the tests in `tests/extras/datasets/spark` are failing, and you are
+> not planning to work on Spark related features, then you can run a reduced
+> test suite that excludes them. Do this by executing the following command:
+> `make test-no-spark`.
```

Comment:
You will need to leave a blank line between "...command:" and the code block, otherwise it will not render.

Reply:
You mean you would like it to render as a separate code block instead of inline?

Reply:
Please. If you have a colon introducing the code, I'd expect to see a separate code block.

Otherwise I'd expect it to read "Do this by executing `make test-no-spark`." without any extra words like `following`. If it's inline, it's not following.

Reply:
Thanks! OK, I'll make this change when I next go into the docs and it'll filter through in due course :)